src/uu/fmt/src/linebreak.rs

```diff
@@ -238,8 +238,8 @@ fn find_kp_breakpoints<'a, T: Iterator<Item = &'a WordInfo<'a>>>(
     let mut active_breaks = vec![0];
     let mut next_active_breaks = vec![];
 
-    let stretch = (args.opts.width - args.opts.goal) as isize;
-    let minlength = args.opts.goal - stretch as usize;
+    let stretch = args.opts.width - args.opts.goal;
+    let minlength = args.opts.goal - stretch;
```

Comment:
This line can overflow, for example with goal=1 and width=75. But I think it's outside the scope of this PR, so I will open a ticket for it.

Reply:
Oh, I see you already mentioned this issue above.

Reply:
Agreed. I wanted to add a test for small "goals" and fixing the calculated widths (e.g., for `-g 2` the width should be 12 according to the documentation) and ran into this bug. I think a saturated subtraction makes sense but wasn't getting the same results as `fmt -g...` and figured I'd leave it for another PR